### PR TITLE
Refactor: Split LogRow component

### PR DESCRIPTION
**Pull request by aocenas**
This moves the row message content to separate component. Main point was I thought I could make toggling labels or time more snappy but that helped to shave only <20%,  as the main bottleneck is the DOM work in this case, and this improvement isn't something very noticeable for the user. In case of 3k rows this means about a 1-2 seconds but still not something user will write home about.

On the other hand I felt like splitting the component makes still sense as it was getting pretty big anyway.

This also adds the styleFactory wrapper as with thousands of rows this actually makes a bit of difference here.

### CODE REVIEW

**Review by kaydelaney — APPROVED** [review]
LGTM

File: packages/grafana-ui/src/components/Logs/LogRowMessage.tsx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+      {props.children}
+      <span
+        className={cx([style, 'logs-row__field-highlight--icon', 'fa fa-signal'])}
+        onClick={() => onClick(props.children)}
+      />
+    </span>
+  );
+};
+
+const logRowStyles = css`
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Logs/LogRowMessage.tsx]: Should this be moved into `stylesFactory`?

File: packages/grafana-ui/src/components/Logs/LogRowMessage.tsx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    const styles = showContext ? cx(logRowStyles, getLogRowWithContextStyles(theme).row) : logRowStyles;
+
+    return (
+      <div
+        className={cx([style.logsRowMessage])}
+        onMouseEnter={this.onMouseOverMessage}
+        onMouseLeave={this.onMouseOutMessage}
+      >
+        <div
+          className={css`
```
Comment (hugohaggmark) [inline packages/grafana-ui/src/components/Logs/LogRowMessage.tsx]: Looks to be the same style as the one commented above?

**Review by hugohaggmark — APPROVED** [review]
LGTM, left some nitpick/minor comments.

**Pull request merged**